Bottom: 7909267cb2e513a0971a86bfe3a391950545739d
Top:    fb5d321b925392980f5d2671de1221db8f2f9446
Author: scott snyder <sss@karma>
Date:   2020-04-18 23:25:40 -0400

Refresh of buttersafe

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Buttersafe.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Buttersafe.java
index 26cd839..c06eec6 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Buttersafe.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/Buttersafe.java
@@ -12,7 +12,7 @@ public class Buttersafe extends ArchivedComic {
 
 	@Override
 	public String getComicWebPageUrl() {
-		return "http://www.buttersafe.com";
+		return "https://www.buttersafe.com";
 	}
 
 	@Override
@@ -37,7 +37,7 @@ public class Buttersafe extends ArchivedComic {
 
 	@Override
 	protected String getArchiveUrl() {
-		return "http://www.buttersafe.com/archive/";
+		return "https://www.buttersafe.com/archive/";
 	}
 
 	@Override
@@ -51,7 +51,7 @@ public class Buttersafe extends ArchivedComic {
 		String final_str = null;
 		String final_title = null;
 		while((str = reader.readLine()) != null) {
-			int index1 = str.indexOf("http://buttersafe.com/comics/");
+			int index1 = str.indexOf("https://www.buttersafe.com/comics/");
 			if (index1 != -1) {
 				final_str = str;
 				final_title = str;
@@ -59,7 +59,7 @@ public class Buttersafe extends ArchivedComic {
 		}
 		final_str = final_str.replaceAll(".*img src=\"","");
 		final_str = final_str.replaceAll("\".*","");
-		final_title = final_title.replaceAll(".*http://buttersafe.com/comics/", "");
+		final_title = final_title.replaceAll(".*https://buttersafe.com/comics/", "");
 		final_title = final_title.replaceAll("\".*", "");
 		final_title = "Buttersafe: " + final_title.substring(11, final_title.length());
 		final_title = final_title.replaceAll(".jpg", "");
